OrbitGl/App.cpp

```diff
@@ -612,12 +610,9 @@ void OrbitApp::ClearCapture() {
   set_selected_thread_id(-1);
   SelectTextBox(nullptr);
 
-  // Trigger deallocation of previous sampling related data.
-  // TODO(kuebler): Investigate why selected process needs to be passed here.
-  auto empty_sampling_profiler = std::make_shared<SamplingProfiler>(GetSelectedProcess());
-  AddSamplingReport(empty_sampling_profiler, nullptr);
```

Comment:
Why it was removed?

Reply:
Sorry, this was unintended

Reply:
For the record, I now moved the initialization of capture data to OnCaptureStarted and also clear capture data on ClearCapture, so that this is again safe and correct here.